src/coreclr/src/vm/readytoruninfo.cpp

```diff
@@ -60,10 +70,7 @@ MethodDesc * ReadyToRunInfo::GetMethodDescForEntryPoint(PCODE entryPoint)
         return NULL;
 #endif
 
-    TADDR val = (TADDR)m_entryPointToMethodDescMap.LookupValue(PCODEToPINSTR(entryPoint), (LPVOID)PCODEToPINSTR(entryPoint));
-    if (val == (TADDR)INVALIDENTRY)
-        return NULL;
-    return dac_cast<PTR_MethodDesc>(val);
+    return m_pCompositeInfo->TryGetMethodDescForEntryPoint(entryPoint);
```

Comment:
`TryGetMethodDescForEntryPoint` does not really reflect what this does well. The difference between `TryGetMethodDescForEntryPoint` and `GetMethodDescForEntryPoint` is the `TryGetMethodDescForEntryPoint` can be only called on composite R2R info.

Reply:
Good point, renamed in 3rd commit.